tests/conftest.py

```
@@ -247,7 +232,7 @@ class DriverParameterizedTests:
     Helper class for generating fixture params using pytest config opts.
 
     """
-    DEFAULT_DRIVERS = ['psycopg2', 'psycopg2cffi']
+    DEFAULT_DRIVERS = ['psycopg2']
```

Ruff (0.14.0)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)